framework/include/constraints/Constraint.h

```diff
@@ -22,7 +22,7 @@ InputParameters validParams<Constraint>();
 /**
  * Base class for all Constraint types
  */
-class Constraint : public ResidualObject, protected GeometricSearchInterface
+class Constraint : public NeighborResidualObject, protected GeometricSearchInterface
```

Comment:
missing NeighborResidualObject source in this commit?

Reply:
Yea it's in a fixup commit (trying to avoid force pushing to your repo)

Reply:
https://github.com/idaholab/moose/pull/16117/commits/85350e9d2c7f3d54b5d03424d280968f0648ca90

Reply:
Force push should be fine. I stopped working on this branch. It is yours.